service/controller/key/cluster.go

```diff
@@ -29,6 +29,11 @@ const (
 	KubernetesSecurePort = 443
 )
 
+const (
+	KubectlVersion              = "9ccdc9dc55a01b1fde2aea73901d0a699909c9cd"
```

Comment:
It would be nice to have a comment explaining which version this is just so we can check that it is compatible with the Kubernetes version.